pcsx2/gui/MainMenuClicks.cpp

```diff
@@ -412,6 +412,26 @@ void MainEmuFrame::Menu_IsoBrowse_Click( wxCommandEvent &event )
 	AppSaveSettings();		// save the new iso selection; update menus!
 }
 
+void MainEmuFrame::Menu_IsoClear_Click(wxCommandEvent &event)
+{
+	wxDialogWithHelpers dialog(this, _("Confirm clearing ISO list"));
+	dialog += dialog.Heading(_("This will clear your entire ISO list. Continue?"));
+
+	if (SysHasValidState())
+		dialog += dialog.Heading(_("(Your running ISO will remain in the list)"));
```

Comment:
Hmm. Instead of conditionally appending this message, maybe it could be combined with the above message and always be shown e.g. 
```
This will clear the ISO list. If an ISO is running it will remain in the list. Continue?
```
It might be better for translators this way (more flexibility), but I dunno.